GabrieleMeoni/END2END
# Create geographical splitter

Creating a geographical splitter as requested by  Issue #12.

* Patches are split according to a geographical request.
* Classes stratification are kept between train and test. 
* Train and tests are randomly split with the requested percentage.
* Reproducibility of split is kept by fixing the seed.

**How to test it:**

* Load the dataset train and test with the same seed. 
* Check the patches that they have are never the same. 
* To check consistency of the split, I would suggest to print the variables "events_locations_tests_list" and "not_events_locations_tests_list" inside geographical_splitter. They should be the same during the same iteration. 
* Check percentage is the one requested. 
* Check the training starts.
